noxfile.py

```diff
@@ -291,3 +291,27 @@ def wheel(session: nox.sessions.Session):
         "import iris; print(f'{iris.__version__=}')",
         external=True,
     )
+
+
+@nox.session
+def benchmarks(session: nox.sessions.Session):
+    """Run the Iris benchmark runner. Run session with `-- -h` for help.
```

Comment:
I appreciate that both work, but since everywhere else in the documentation uses `-- --help`, it seems slightly inconsistent to use `-- -h` here.

Reply:
```suggestion
    """Run the Iris benchmark runner. Run session with `-- --help` for help.
```